packages/@tailwindcss-upgrade/src/utils/extract-static-plugins.test.ts

```diff
@@ -114,21 +217,67 @@ describe('findStaticPlugins', () => {
     ).toEqual(null)
   })
 
-  test('bails for plugins with options', () => {
+  test('bails on invalid plugin options', () => {
     expect(
       findStaticPlugins(js`
-        import plugin1 from './plugin1'
+        import plugin from './plugin'
+
+        export default {
+          plugins: [
+            plugin({ foo }),
+          ]
+        }
+    `),
+    ).toEqual(null)
+
+    expect(
+      findStaticPlugins(js`
+        import plugin from './plugin'
 
         export default {
-          plugins: [plugin1({foo:'bar'})]
+          plugins: [
+            plugin(),
```

Comment:
I believe the only case where this is valid is if using `plugin.withOptions(…)` — in which case the options are not required. So… I think we can just convert this to `@plugin "./plugin"` yeah?

Reply:
Ah yeah I didn't think this was a valid use case, let me add that.